qiskit/exceptions.py

```diff
@@ -61,6 +61,25 @@
 is not present, but will raise :exc:`OptionalDependencyImportWarning` to let you know about it.
 
 .. autoexception:: OptionalDependencyImportWarning
+
+When experimental features are being used, Qiskit will raise :exc:`ExperimentalWarning`.
+
+.. autoexception:: ExperimentalWarning
+
+
+Filtering warnings
+------------------
+
+Python has built-in mechanisms to filter warnings, described in the documentation of the
+:mod:`warnings` module.  You can use these subclasses in your warning filters from within Python to
+silence warnings you are not interested in.  For example, if you are knowingly using experimental
+features and are comfortable that they make break in later versions, you can silence
+:exc:`ExperimentalWarning` like this:
+
+    import warnings
+    from qiskit.exceptions import ExperimentalWarning
+
+    warnings.filterwarnings("ignore", category=ExperimentalWarning)
```

Comment:
Is this still needed anymore now that #11326 has merged?

Reply:
No, merge commit 0c517fe3d8fe8b3d5187b720cd2c308d5d2f1463 should have fixed it.